File: Stage_0_XR.java
public class Main{

static int hammingDist(String str1, String str2)
{
    int i = 0, count = 0;
    while (i < str1.length())
    {
        if (str1.charAt(i) != str2.charAt(i))
            count++;
        i++;
    }
    return count;
}
     public static void main(String []args){
         String str1 = "XR";
         String str2 = "@Bhushan44825";
          
        System.out.println("Name:Bhushan Wagh \n");
        System.out.println("Email:waghbhusahn1123@gmail.com \n");
        System.out.println("Slack_username:@XR \n");
        System.out.println("Biostack:Software Developement \n");
        System.out.println("Twitter_handle:@Bhushan44825 \n");
        System.out.println(hammingDist (str1, str2));
     }
}

File: Stage_0_XR.php
<html>
<head>
<title>Stage 0 XR</title>
</head>
<body>
<?php
// PHP program to find hamming distance b/w
// Slack_username & Twitter_handler
function hammingDist($Slack_username, $Twitter_handel)
{
    $i = 0; $count = 0;
    while (isset($Slack_username[$i]) != '')
    {
        if ($Slack_username[$i] != $Twitter_handel[$i])
            $count++;
        $i++;
    }
    return $count;
}
    $Slack_username = "XR";
    $Twitter_handel= "@Bhushan44825";
    echo("Name:Bhushan Wagh \n"); 
    echo("Email:waghbhusahn1123@gmail.com \n"); 
    echo("Slack_username:@XR \n");
    echo("Biostack:Software Developement \n"); 
    echo("Twitter_handle:@Bhushan44825 \n");
    echo hammingDist ($Slack_username, $Twitter_handel);
    echo"\n";
     ?>
</body>
</html>